itest/lnd_payment_test.go

```diff
@@ -273,13 +281,15 @@ func testListPayments(ht *lntest.HarnessTest) {
 			req := &lnrpc.ListPaymentsRequest{
 				CreationDateStart: tc.startDate,
 				CreationDateEnd:   tc.endDate,
+				CountTotalPayments: true,
 			}
 			resp := alice.RPC.ListPayments(req)
 
 			if tc.expected {
 				require.Lenf(t, resp.Payments, 1, "req=%v", req)
```

Comment:
also check `resp.TotalNumPayments` is 1 here?

Reply:
Yes, you right. It makes the test more complete. I'll be changing it.

Reply:
done 07bf0a1968280138fbc5496cac3774bdbb89e82e